scripts/reports/count-non-stricken-event-codes-by-year.ts

```diff
@@ -0,0 +1,134 @@
+// usage: npx ts-node --transpile-only scripts/reports/count-non-stricken-event-codes-by-year.ts M071,M074 2021-2022 [includeStricken]  > ~/Desktop/m071s-and-m074s-filed-2021-2022.csv
```

Comment:
Fix filename, also don't pipe the output to a CSV.

Reply:
Blargh - sorry, pipe removed.  What's wrong with the filename?

Reply:
See the previous comment - this report can optionally count stricken docket entries, so we don't need to specify "non-stricken" in the filename.